js/src/components/shipping-rate-section/minimum-order-card/edit-minimum-order-form-modal.js

```diff
@@ -13,12 +14,33 @@ import AppInputPriceControl from '.~/components/app-input-price-control';
 import VerticalGapLayout from '.~/components/vertical-gap-layout';
 import AppCountrySelect from '.~/components/app-country-select';
 
-const EditMinimumOrderModal = ( props ) => {
-	const { countryOptions, value, onChange, onRequestClose } = props;
+/**
+ * Display the edit minimum order modal that is wrapped in a Form.
+ *
+ * When users submit the form, `props.onRequestClose` will be called first, and then followed by `props.onSubmit`.
+ * If we were to call `props.onSubmit` first, it may cause some state change in the parent component and causing this component not to be rendered,
+ * and when `props.onRequestClose` is called later, there would be a runtime React error because the component is no longer there.
+ *
+ * @param {Object} props Props.
+ * @param {Array<string>} props.countryOptions Array of country codes options, to be used as options in AppCountrySelect.
+ * @param {Object} props.initialValues Initial values for the form.
+ * @param {Array<string>} props.initialValues.countries Array of selected country codes.
+ * @param {string} props.initialValues.currency Selected currency.
+ * @param {number} props.initialValues.threshold Threshold value.
+ * @param {function()} props.onRequestClose Callback to close the modal.
+ * @param {function(Object)} props.onSubmit Callback when the form is submitted, with the form value.
+ */
+const EditMinimumOrderFormModal = ( props ) => {
+	const {
+		countryOptions,
+		initialValues,
+		onRequestClose = noop,
+		onSubmit = noop,
+	} = props;
 
```

Comment:
:nail_care: Same as https://github.com/woocommerce/google-listings-and-ads/pull/1410/files#r845530511

Reply:
Same fix in [`11ca55c` (#1410)](https://github.com/woocommerce/google-listings-and-ads/pull/1410/commits/11ca55c460539722dc72b4abb502f12570d55910).